app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tidepool-uploader",
   "productName": "tidepool-uploader",
-  "version": "2.18.0",
+  "version": "2.18.0-dexcom-touchscreen.3",
```

Comment:
Should this version bump to `2.19.0-dexcom-touchscreen.3`?

Reply:
I've built a test version for Eric to try out while this is in code review, and he discovered some issues. Once we get these resolved I'll remove the test version strings and request re-review, thanks!